fecgov/fec-cms
# [WIP] Feature upgrade wagtail

## Summary

This PR upgrades our CMS from Wagtail v2 to Wagtail v4. 

### Required reviewers

All of frontend and content teams

## Impacted areas of the application

General components of the application that this PR will affect:

-  Entire FEC-CMS Wagtail

## Screenshots

TBD

## How to test

- See upgrade [documentation](https://docs.google.com/document/d/1qvdv5LxImKbzLrc9Y_swN7Ag9w7LXQRiyVJJIBVTzfM/edit#heading=h.wk3f01b7mk6e)
